include/aspect/mesh_deformation/fastscape.h

```diff
@@ -0,0 +1,725 @@
+       */
+      double left_flux;
+
+      // FastScape erosional parameters //
+      /**
+       * Drainage area exponent for the stream power law (m variable in FastScape surface equation).
+       */
+      double m;
+
+      /**
+       * Slope exponent for the steam power law (n variable in FastScape surface equation).
+       */
+      double n;
+
+      /**
+       * Slope exponent for multi-direction flow, where 0 is uniform, and 10 is steepest descent. (-1 varies with slope)
+       * (p variable in FastScape surface equation).
+       */
+      double p;
+
+      /**
+       * Bedrock deposition coefficient. Higher values deposit more sediment
+       * inside the domain. (G variable in FastScape surface equation).
+       */
+      double g;
+
+      /**
+       * Sediment deposition coefficient. Higher values deposit more sediment inside the domain.
+       * When set to -1 this is identical to the bedrock value.
+       * (G variable in FastScape surface equation applied to sediment).
+       */
+      double gsed;
+
+      /**
+       * Bedrock river incision rate for the stream power law
+       * (meters^(1-2m)/yr, kf variable in FastScape surface equation).
+       */
+      double kff;
+
+      /**
+       * Sediment river incision rate for the stream power law (meters^(1-2m)/yr).
+       * When set to -1 this is identical to the bedrock value.
+       * (kf variable in FastScape surface equation applied to sediment).
+       */
+      double kfsed;
+
+      /**
+       * Bedrock transport coefficient for hillslope diffusion (m^2/yr, kd in FastScape surface equation).
+       */
+      double kdd;
+
+      /**
+       * Bedrock transport coefficient for hillslope diffusion (m^2/yr). When set to -1 this is
+       * identical to the bedrock value.
+       * (kd in FastScape surface equation applied to sediment).
+       */
+      double kdsed;
+
+      // FastScape marine parameters //
+      /**
+       * Fastscape sea level (m), set relative to the ASPECT surface where
+       * a sea level of zero will represent the maximum Y (2D) or Z (3D) extent
+       * inside ASPECT.
+       */
+      double sea_level;
+
+      /**
+       * Parameters to set an extra erosional base level
+       * on the ghost nodes that differs from sea level.
+       */
+      bool use_fixed_erosional_base;
+
+      /**
+       * Height of the extra erosional base level.
+       */
+      double h_erosional_base;
+
+      /**
+       * Surface porosity for sand.
+       */
+      double p1;
+
+      /**
+       * Surface porosity for silt.
+       */
+      double p2;
+
+      /**
+       * Sands e-folding depth for exponential porosity law (m).
+       */
+      double z1;
+
+      /**
+       * Silts e-folding depth for exponential porosity law (m).
+       */
+      double z2;
+
+      /**
+       * Sand-silt ratio
+       */
+      double r;
+
+      /**
+       * Averaging depth/thickness for sand-silt equation (m).
+       */
+      double l;
```

Comment:
same issue here. single-letter variable names are just very difficult to understand in code if they are not purely local counting indices.